package.py

```diff
@@ -15,6 +15,7 @@
 # - addon with specified version range must exist to be able to use this addon
 ayon_required_addons = {
     "core": ">0.4.4",
+    "applications": ">1.0.1"
```

Comment:
Technically, you don't need that new applications. You can define cinema4d in additional applications...

Reply:
Additional applications don't seem to allow to specify a "host" name/integration - so I guess that wouldn't work?

Reply:
Fixed with https://github.com/ynput/ayon-cinema4d/pull/1/commits/ce1217d9e7f9abc61c6e11e029aae2efe9f85553